x-pack/plugins/security_solution/server/lib/detection_engine/rule_management/api/rules/bulk_actions/route.ts

```diff
@@ -122,18 +130,29 @@ const buildBulkResponse = (
     deleted = [],
   }: {
     isDryRun?: boolean;
-    errors?: BulkActionError[];
+    errors?: BulkActionError[] | BulkEditError[];
     updated?: RuleAlertType[];
     created?: RuleAlertType[];
     deleted?: RuleAlertType[];
   }
 ) => {
+  const skipped = (
+    errors.filter((error) => {
+      if (isBulkEditError(error)) {
+        return Object.values(BulkEditSkipReason).includes(error.message as BulkEditSkipReason);
+      }
+      return false;
+    }) as BulkEditError[]
+  ).map(({ rule }) => rule) as RuleAlertType[];
```

Comment:
`skipped` rules are returned from the `rules_client` as errors, and we filter them out through their error message as listed in the `BulkEditSkipReason`. 

My doubt here is: if we do it this way, the skipped rules as returned by the endpoint won't have all the rule information with it cause the `rules_client` returns rules that had an error only with their `name` and `id`. My opinion is that that is enough to report how many and which rules were skipped, but it does make the response of the API endpoint inconsistent, since rules listed in `updated`, `created` and `deleted` have all the rules properties within it, while `skipped` has only `name` and `id`.

The `rules_client` can be edited to return the full rule information. I tried it and can be done, but it implies some not-ideal typing changes which in my opinion are not worth doing for this use case.

Open to opinions though.